madminer/utils/morphing.py

```diff
@@ -301,7 +420,7 @@ def optimize_basis(
         # Normal output
         return best_basis
 
-    def calculate_morphing_matrix(self, basis=None):
+    def calculate_morphing_matrix(self, basis=None, gd = None, gp = None, gs = None):
```

Comment:
Even if we have not officially added Black as the project format checker, we should try to follow its style guide.

```suggestion
    def calculate_morphing_matrix(self, basis=None, gd=None, gp=None, gs=None):
```

Reply:
Thank you, I didn't run black in this commit but will run it before next commit.